assets/prefabs/potion/ExplosivePotion.prefab

```diff
@@ -0,0 +1,28 @@
+{
+    "parent": "engine:iconItem",
+    "Item": {
+        "icon": "Potions:HarmPotionBottle",
+        "stackId": "Potions:explosivePotion",
+        "consumedOnUse": true
+    },
+    "DisplayName": {
+        "name": "Explosive Potion",
+        "description": "If consumed, explodes the player."
+    },
+    "Potion": {
+        "hasGenome": false,
+        "effects": [
+            {
+                "effect": "EXPLOSIVE",
+                "magnitude": 2,
+                "duration": 1
+            }
+        ]
+    },
+    "ItemHelp": {
+        "category": "Potions",
+        "paragraphText": [
+             "If consumed, kills the user."
```

Comment:
Suspecting this line may not have been updated from origin text? Since it differs from the DisplayName text :-)